app/workers/import_worker.rb

```diff
@@ -0,0 +1,15 @@
+class ImportWorker
+  include Sidekiq::Worker
+
+  def perform(device, json_file)
```

Comment:
Since file is uploaded via import request, it should be stored temporarily. Why not we just pass path to worker and read file from worker then? Without json parsing.